### ISSUE TITLE
linting paper-input gives misleading warning

### ISSUE THREAD

**Issue by stramel** [Polymer/polymer-linter#153]
`polymer lint -r polymer-2-hybrid --fix` applies `slot="input"` to `<span id="teamplte-placeholder">` which gets replaced by content that has the appropriate `slot="input"`. This results in a misleading warning:

```sh
$ polymer lint -r polymer-2-hybrid


    <paper-input-container id="container"
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        no-label-float="[[noLabelFloat]]"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        always-float-label="[[_computeAlwaysFloatLabel(alwaysFloatLabel,placeholder)]]"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        auto-validate$="[[autoValidate]]"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        disabled$="[[disabled]]"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        invalid="[[invalid]]">
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

paper-input.html(158,4) warning [content-to-slot-usage-site] - Deprecated <content>-based distribution into <paper-input-container>. Must use the `slot` attribute for named distribution."


Found 1 warning. 1 can be automatically fixed with --fix.
```

There was also, a few other warnings that I'm looking at regarding content to slot usages in the tests but they seem correct.

**Comment by rictic** [comment]
Ah, yeah, there paper-input is doing something too clever for the linter to understand. I think the solution would be to add `content-to-slot-usage-site` to paper-input's ignored warnings in its polymer.json file.

**Comment by stramel** [comment]
Perhaps some linter comment directives in the future? lol

**Comment by FredKSchott** [comment]
@stramel that sounds familiar :) we have this sitting around but not prioritized atm https://github.com/Polymer/polymer-analyzer/blob/master/design/comment-scanning.md

**Comment by stramel** [comment]
I remember that you wrote something up and had a initial implementation that stalled out. Just figured I would hint at it 😉
